### ISSUE TITLE
Documentation how to release macOS catalyst apps on App Store

### ISSUE THREAD

**Issue by rstolpe** [dotnet/maui#12455]
### Description

The iOS documentation are great and I got it working right away, but the documentation for mac catalyst applications release as signed on Apple Store are non existing and I don't get it to work.
please provide proper documentation

### Public API Changes

none

### Intended Use-Case

All users

**Comment by mattleibow** [comment]
@davidbritch are we missing mac catalyst docs?

**Comment by davidbritch** [comment]
@mattleibow Yes.

**Comment by mattleibow** [comment]
Should we move this to the docs repo or do we have an issue already?

**Comment by rstolpe** [comment]
@mattleibow @davidbritch I got some things figured out that was missing in the info.plits and Entitlements.
I have not successed to publish it  as I get the following error now:

Asset validation failed (90259)
Bad Bundle Executable. You must include a valid CFBundleExecutable key in your bundle's information property list file. (ID: a630ade7-1660-45f4-b69b-xxxx)

I have added it in the info.plits but no success with that either.
Same code works to publish to iOS.

Also when you compile it as arm-64 it won't start or it do start but it closes / crashes.

You can't even create arm-x64 when you debug it only works with x64.

Same with new project.

**Comment by rstolpe** [comment]
@mattleibow @davidbritch where is the doc repo located?
I got this figured out now after working around a bunch of bugs etc.

Have everything written in a guide ready to be uploaded tomorrow.

**Comment by rstolpe** [comment]
Uploaded the guide in a Feature request.
https://github.com/dotnet/maui/issues/12504

If you have any Q just ask in that issue and I'll answer it.

**Comment by cdavidyoung** [comment]
So I have worked through https://github.com/dotnet/docs-maui/issues/1187 but have yet to get it to work.  Even if I get it to work my question, why is this so much harder and convoluted than iOS deployment to the App Store?  With the latest VS2022 it seems to have gotten even easier because I was able to upload my iOS app to the App Store within VS.  However, the same app for Maccatalyst is stuck in this provisioning purgatory.  From the App Store Connect perspective is there really much difference between submitting Maccatalyst and iOS?
